src/client/common/errors/types.ts

```diff
@@ -22,6 +22,15 @@ export class WrappedError extends BaseError {
             this.stack = `${new Error('').stack}${EOL}${EOL}${originalException.stack}`;
         }
     }
+
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    public from(err: any) {
```

Comment:
Please change this to a static function & return the error instead of throwing.

public static from (error: any){
	if (error instanceOf BaseError){
		return error;
	} else {
		return new WrappedError(...)
	}
}

Reply:
`from` by general convention returns something and doesn't throw exceptions.